src/validators/RegularExpressionValidator.php

```
@@ -33,19 +37,30 @@ class RegularExpressionValidator extends Validator
         }
-        if ($this->message === null) {
-            $this->message = Yii::t('yii', '{attribute} is invalid.');
+
+        $this->message ??= Yii::t(
+            'yii',
+            '{attribute} is invalid.',
+        );
+
+        if ((Yii::$app->useJquery ?? false) && !$this->clientScript instanceof ClientValidatorScriptInterface) {
+            $this->clientScript ??= ['class' => RegularExpressionValidatorJqueryClientScript::class];
+            $this->clientScript = Yii::createObject($this->clientScript);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'init'. (undefined)

(StaticAccess)